posts/2020/08/but-what-are-traitlets.md

```diff
@@ -0,0 +1,305 @@
+<!--
+.. title: But What are Traitelts ?
```

Comment:
I'd think a more descriptive title like `Traitlets - an introduction & use in Jupyter config management" or something else like that would be good.